lib/posexional/file.ex

```diff
@@ -49,8 +49,8 @@ defmodule Posexional.File do
       |> manage_counters
       |> get_lines(values)
       |> Stream.map(fn line ->
-        IO.write(handle, line)
-        IO.write(handle, separator)
+        IO.binwrite(handle, line)
```

Comment:
forse forse visto che lavoriamo con roba in utf-8 preferisco l'altro approccio, ovvero specificare `:utf8` come opzione quando apriamo il file e usare `IO.write`

Reply:
ma ad ania va mandato il latin1 ovvimante

Reply:
non credo proprio.. mi auguro sia tutto utf8 altirmenti mi ammazzo.. e poi se fai IO.binwrite di una stringa elixir quello che ti esce è comunque utf8

Reply:
ah ma poi ho visto adesso che è outdated, l'hai modificato nella commit dopo, vado a vedere

Reply:
Abbiamo sempre mandato i file in latin1. 
``` 
ile -i crash_den_20201027060115
crash_den_20201027060115: text/plain; charset=iso-8859-1
``` 
Non lo modificherei, dato che è molto delicato e nella documentazione non cè scritto niente

Reply:
cosi a occhio direi che il charset non c'entra con l'encoding ma indago un attimo

Reply:
o cristo, iso-8859-1 oltre a essere un charset è anche un encoding, e sui caratteri non ascii è diverso da utf-8.. quello che non mi è per niente chiaro è dove viene deciso l'encoding nel codice.. @mbusi help please

Reply:
per la precisione.. 
```
[pts/6:zuliani@Zuliani-ThinkPad-T580 zuliani/Downloads ]% uchardet crash_den_20201027060115 
ISO-8859-2
```

Reply:
il default di file open è latin1
```
latin1
The default encoding. Bytes supplied to the file, that is, write/2 are written "as is" on the file. Likewise, bytes read from the file, that is, read/2 are returned "as is". If module io(3) is used for writing, the file can only cope with Unicode characters up to code point 255 (the ISO Latin-1 range).
```

Reply:
o cristo santo schifoso.. che MERDAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
ok allora lasciamo cosi che non si mai con ania merda